share/availability/test/testing.go

```diff
@@ -17,8 +17,8 @@ import (
 	record "github.com/libp2p/go-libp2p-record"
 	mocknet "github.com/libp2p/go-libp2p/p2p/net/mock"
 	"github.com/stretchr/testify/require"
-	"github.com/tendermint/tendermint/pkg/da"
 
+	"github.com/celestiaorg/celestia-app/pkg/da"
```

Comment:
```suggestion
	"github.com/celestiaorg/celestia-app/pkg/da"
	
```

Reply:
Ou of curiosity, what motivates this suggestion? Does celestia-node prefer to separate some imports with newlines? If so, which imports?

Related: it looks like https://golangci-lint.run/usage/linters/#goimports doesn't have a config option for newlines between particular import statements but it may speed up reviews if maintainers could rely on a linter for this

Reply:
Reverted this change b/c it fails lint https://github.com/celestiaorg/celestia-node/pull/1147/commits/44c8d92cc672c9cd04dc5d939a00687f55d4bc32